src/content/app/species-selector/state/species-selector-api-slice/speciesSelectorApiSlice.ts

```diff
@@ -46,12 +47,7 @@ export type GenomesSearchBySpeciesTaxonomyIdRequestParams = {
 // either because of the temporal dead zone that occurs otherwise (though typescript doesn't complain)
 // or due to some bundler nonsense
 const transformGenomesSearchResponse = (response: SpeciesSearchResponse) => {
-  response.matches.forEach((match) => {
-    match.common_name = match.common_name
-      ? upperFirst(match.common_name)
-      : match.common_name;
-  });
-
+  response.matches = response.matches.map(formatGenomeData);
```

Comment:
Do the comments above still apply here?

Reply:
Sorry, @imransl ; I will have to change this PR to remove all the upper-casing that it does, and to revert the upper-casing for the species search api. I'm going to mark this PR as a draft. Meanwhile, could you take a look at [undo all the upper-casing](https://github.com/Ensembl/ensembl-client/pull/1054)?